bgc_derivation/derivations/Derivation.m

```diff
@@ -5,29 +5,36 @@
     
     properties
         variables;  % Variables extracted from source data
+		coeffs;  % predeployment coeffs 
         default_glob;  % Default glob pattern used for selecting data files
     end
     properties (Abstract, Constant = true)
         equation;  % Underlying equation function
     end
     methods
-        function self = Derivation(variables)
+        function self = Derivation(variables, coeffs)
             % Constructor for Derivation objects. Accepts a pre-configured
             % Map of variables or can use add_ methods to ingest data.
             %
-            % derivation = Derivation([variables])
+            % derivation = Derivation([variables, coeffs ])
             % where
             %  variables is a pre-populated name-value Map of variables.
             %
             
             self.default_glob = '**/*.nc';
             
-            if nargin == 1
-                self.variables = variables;
-            else
-                self.variables = containers.Map;
-            end
+			switch nargin
```

Comment:
The indentation looks a bit off here.

Reply:
done. very strange. It was not visible in MATLAB